### DISCUSSION TITLE
Logic Analyser output to PulseView/Sigrok?

### DISCUSSION THREAD

**Discussion by tony1tf** [fhdm-dev/scoppy#77] [General]
Hi

Looks like a great project, but the logic analyser side would be greatly enhanced if files could be generated that are compatible with PulseView/Sigrok. This can use decoders written in Python to display any serial data streams (I2C, UART etc). On the EDSAC replica project at The National Museum of Computing we use it to display the serial data streams with our own Python code.
Tony

**Comment by fhdm-dev** [comment]
Hi,

Yep, I can see how that would be useful. The app can currently export the sample data as a CSV file but there's nothing specific to Sigrok. I'll look into it.

**Comment by tony1tf** [comment]
i can probably write some code to convert a csv file to Sigrok format -
I'll look into it.

Tony
